app/api/files/jsRoutes.js

```diff
@@ -76,6 +86,11 @@ export default app => {
           socket(req).emit('documentProcessed', newEntity.sharedId);
         });
       }
+
+      if (req.body.email) {
+        mailer.send(JSON.parse(req.body.email));
```

Comment:
Should this somehow be wrapped in a try - catch scenario? What happens if the `email` json is incorrectly formed? The parse here will break and we will end up with an un-responded `res`.  Should there be more `next(err)` contemplated here?

Perhaps, even more critical, mailer is a Promise, which is been implemented here without an await, then or catch.  Is this prone to uncatched rejection scenarios?

Reply:
About what happens if the email JSON is malformed, I think that if we add validation to the request is to trust the structure of the data we receive, if we don't trust that structure we lack a lot of try-catch in all the app, but I guess the mailer can fail because of wrong configuration in settings, that I don't think we can have it validated.